src/modules/admin/AdminPage.vue

```diff
@@ -51,6 +51,15 @@
           <CalloutSummary :callout="latestCallout" footer />
         </a>
       </template>
+      <template v-else>
+        <AppHeading>{{ t('adminDashboard.latestCallout.title') }}</AppHeading>
+        <p class="block bg-white p-4 mt-4 mb-8 rounded">
+          No currently active callouts.
```

Comment:
Needs to be localised

Reply:
Hup, done, added a couple of new entries. Should I mark somewhere when there's new locale strings added?